src/modules/organizations/actions.js

```diff
@@ -41,15 +41,15 @@ export function fetchById(organizationId) {
 
 export function fetchSuccess(organizations) {
   return (dispatch) => {
-    const formatted = organizations.map(o => _.pick(o, ['id', 'name', 'picture', 'admin_id', 'account']))
+    const formatted = organizations.map(o => _.pick(o, ['id', 'name', 'picture', 'admin_id', 'account', 'plan']))
     dispatch(oActions.fetchSuccess(formatted))
   }
 }
 
-export function create(name) {
+export function create({name, planId}) {
```

Comment:
Kind of a weird API here; taking an object with signature {name, planId} just to use an object later with signature {name, plan}. Why not just take the organization object directly in the create.